### PR TITLE
feat(explorer): add title metadata to pages

### PR DESCRIPTION
**Pull request by karooolis**

### CODE REVIEW

File: packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/page.tsx [hunk 1]
```diff
@@ -1,5 +1,10 @@
+import { Metadata } from "next";
 import { Explorer } from "./Explorer";
 
+export const metadata: Metadata = {
+  title: "Explore",
```
Comment (holic) [inline packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/page.tsx:5]: this is fine for now but would be nice to have this be something like 

> Explore - 0x1234...5678 on Redstone

(I don't think we need "World Explorer" in every title)

File: packages/explorer/src/app/(explorer)/layout.tsx [hunk 2]
```diff
@@ -17,7 +17,10 @@ const jetbrains = JetBrains_Mono({
 });
 
 export const metadata: Metadata = {
-  title: "World Explorer",
+  title: {
+    default: "World Explorer",
```
Comment (holic) [inline packages/explorer/src/app/(explorer)/layout.tsx:21]: World Explorer or World**s** Explorer?

**Review by holic — APPROVED** [review]

File: packages/explorer/src/app/(explorer)/layout.tsx (hunk 2 above)
Comment (karooolis) [reply packages/explorer/src/app/(explorer)/layout.tsx:21]: Yeah good observation, updated to "Worlds"

File: packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/page.tsx (hunk 1 above)
Comment (karooolis) [reply packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/page.tsx:5]: Yeah agreed, would be nice once we have world names available. And removed "World Explorer" from every title.

**Review by holic — APPROVED** [review]

**Pull request merged**